source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -543,6 +543,16 @@ AdaptiveNamespaceStart
         All = Left | Right | Up | Down
     } BleedDirection;
 
+    [version(NTDDI_WIN10_RS1), flags]
+#ifdef ADAPTIVE_CARDS_WINDOWS
+    [contract(InternalContract, 1)][internal]
+#endif
+    typedef[v1_enum] enum InputNecessityIndicators {
```

Comment:
>InputNecessityIndicators [](start = 26, length = 24)

Code looks good. I was hoping that we'd have another round of spec review once the showcard stuff was ironed out. I don't like this name, and wonder if we can't come up with something better. That said, this PR shouldn't be blocked on that... Please let me know once this branch is merged and I'll file a bug. :)